public/app/features/playlist/endpoints.gen.ts

```diff
@@ -0,0 +1,529 @@
+import { baseAPI as api } from './baseAPI';
+const injectedRtkApi = api.injectEndpoints({
+  endpoints: (build) => ({
+    getApiResources: build.query<GetApiResourcesApiResponse, GetApiResourcesApiArg>({
+      query: () => ({ url: `/apis/playlist.grafana.app/v0alpha1/` }),
+    }),
+    listPlaylists: build.query<ListPlaylistsApiResponse, ListPlaylistsApiArg>({
```

Comment:
How did you get these endpoint names (without the namespace stuff), manual search + replace?

Reply:
Yeah - I manually edited the OpenAPI spec and then generated from that https://github.com/grafana/grafana/pull/87545/commits/0e2a38c4e492a2c66a3357ac8720ef21d3864a75